projects/canopy/src/lib/spacing/spacing.module.ts

```diff
@@ -1,11 +1,12 @@
 import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
 
-import { LgMarginModule } from './margin/margin.module';
-import { LgPaddingModule } from './padding/padding.module';
+import { LgMarginModule } from './margin';
```

Comment:
How weird, I'm pretty sure I removed the spacing module in favour of separate modules for each directive 🤔 are you sure you are up to date?

Reply:
I think I'm wrong as I'm seeing the same in master. I was sure I did it but I must be confusing it with the `canopy.module`. Let's keep it as is for now 👍